perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -645,7 +648,7 @@ class YCSBExecutor(object):
 
   def __init__(self, database, parameter_files=None, **kwargs):
     self.database = database
-    self.loaded = False
+    self.loaded = True
```

Comment:
Why change this to True?

Reply:
this was for a workaround so i could do subsequent tests with the same loaded data, I changed this back to original.